src/features/jumpNav.tsx

```diff
@@ -641,9 +598,38 @@ export const toggleFeature = (flag: boolean) => {
+    unloads.add(addCommand({label: "Toggle Linked Refs",callback: toggleLinkedRefs,},extensionAPI));
+    unloads.add(addCommand({label: "Toggle Unlinked Refs",callback: toggleUnlinkedRefs,},extensionAPI));
+    unloads.add(addCommand({label: "Toggle References to page level",callback: toggleReferenceParents,},extensionAPI));
+    unloads.add(addCommand({label: "Expand Reference children",callback: expandReferenceChildren,},extensionAPI));
+    unloads.add(addCommand({label: "Collapse Reference children",callback: collapseReferenceChildren,},extensionAPI));
+    unloads.add(addCommand({label: "Copy block ref",callback: copyBlockRef,},extensionAPI));
+    unloads.add(addCommand({label: "Copy block ref as alias",callback: copyBlockRefAsAlias,},extensionAPI));
+    unloads.add(addCommand({label: "Expand current block tree",callback: expandCurrentBlockTree,},extensionAPI));
+    unloads.add(addCommand({label: "Collapse current block tree",callback: collapseCurrentBlockTree,},extensionAPI));
+    unloads.add(addCommand({label: "Insert block above",callback: insertBlockAbove,},extensionAPI));
+    unloads.add(addCommand({label: "Insert block below",callback: insertBlockBelow,},extensionAPI));
+    unloads.add(addCommand({label: "Go up a block",callback: goUpBlock,},extensionAPI));
+    unloads.add(addCommand({label: "Go down a block",callback: goDownBlock,},extensionAPI));
+    unloads.add(addCommand({label: "Go to parent block",callback: goToParentBlock,},extensionAPI));
+    unloads.add(addCommand({label: "Delete block",callback: delBlock,},extensionAPI));
+    unloads.add(addCommand({label: "Toggle Block View type",callback: toggleBlockViewType,},extensionAPI));
+    unloads.add(addCommand({label: "Replace last reference before cursor with text and alias",callback: replaceLastReferenceWithTextAndAlias,},extensionAPI));
+    unloads.add(addCommand({label: "Apply Children of last reference before cursor as text",callback: applyChildrenOfLastReferenceAsText,},extensionAPI));
+    unloads.add(addCommand({label: "Replace last reference before cursor with original + bring nested items along",callback: replaceLastReferenceWithOriginal,},extensionAPI));
+    unloads.add(addCommand({label: "Paste block with children as references",callback: pasteBlockWithChildrenAsReferences,},extensionAPI));
+    unloads.add(addCommand({label: "Expand/Collapse block tree to a certain level, specified by the following numeric key press",callback: expandCollapseBlockTree,},extensionAPI));
+    unloads.add(addCommand({label: "Align left",callback: alignLeft,},extensionAPI));
+    unloads.add(addCommand({label: "Center",callback: center,},extensionAPI));
+    unloads.add(addCommand({label: "Align right",callback: alignRight,},extensionAPI));
+    unloads.add(addCommand({label: "Justify",callback: justify,},extensionAPI));
+    unloads.add(addCommand({label: "Toggle Queries",callback: toggleQueries,},extensionAPI));
```

Comment:
Another way to make this readable is to make this more DRY (don't repeat yourself):

```typescript
const commands = [
  {label: "Justify", callback: justify},
  {label: "Toggle Queries", callback: toggleQueries},
  // ... etc.
];

commands.forEach((cmd) => 
  unloads.add(addCommand(cmd, extensionAPI))
);
```